tasks/github_tasks.py

```diff
@@ -184,6 +186,86 @@ def send_rate_limit_info_datadog(_, pipeline_id):
 
 
 @task
+def publish_comment_on_pr(_, branch_name, pipeline_id, job_name, executed_test: bool):
+    from .libs.common.github_api import GithubAPI
+
+    jobs_regex = re.compile(r"  - ([a-z-A-Z_0-9]*)")
+    pipeline_id_regex = re.compile(r"pipeline ([0-9]*)")
+    if branch_name == "" or job_name == "":
+        return
+    print("Branch name: ", branch_name)
+    print("Pipeline id: ", pipeline_id)
+    print("Job name: ", job_name)
+    print("Executed test: ", executed_test)
```

Comment:
```suggestion
    print("Branch name:", branch_name)
    print("Pipeline id:", pipeline_id)
    print("Job name:", job_name)
    print("Executed test:", executed_test)
```

Reply:
Actually was more for debugging purpose. I will delete it